pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
+> More information: <https://https://developer.gnome.org/dconf>.
```

Comment:
```suggestion
> More information: <https://developer.gnome.org/dconf>.
```

Reply:
Whoops

Reply:
Done it for the other one as well, I have added the suggestion for it